iso15118/secc/transport/tcp_server.py

```diff
@@ -112,6 +114,11 @@ async def server_factory(self, tls: bool) -> None:
             f"port {port}"
         )
 
+        if tls:
+            self.tls_ready_event.set()
+        else:
+            self.tcp_ready_event.set()
+
```

Comment:
Same comment regarding udp server as mentioned below. Wouldn't it suffice if we pass in the ready_event Event through the start_tls() or start_no_tls() method - as the state is not required after this - so making it a member variable would not be required. What do you think?

Reply:
I thought it could be used in the future to notify when a server failed. It's not needed for now.